.mbedignore

```diff
@@ -1 +0,0 @@
-wifi-x-nucleo-idw01m1/*
```

Comment:
Is the build failure now fixed?
This driver did used to refer some ST defined pin names so it did not build on other that ST platforms.

Reply:
Well, this was in because both ESP8266 & IDW0XX1 expansion boards used external library `ATParser` and this caused conflicts when both were compiled.

Reply:
That was the initial assumption. Later, these drivers were fixed but we realised that IDW0XX1 was referring to hard coded reset pin name.
I need to test whether this is still true.

Reply:
This should be fixed in the lastest version of the WiFi driver's `master` branch.